src/lib/forms-schema/index.ts

```diff
@@ -209,11 +209,11 @@ export const WorkflowEventSchema = Joi.object().keys({
           uri: Joi.number().required(),
           label: Joi.string().required(),
         }),
-        actionDefinition: Joi.object().keys({
+        actionDefinition: Joi.object().optional().keys({
           uri: Joi.number().required(),
           label: Joi.string().required(),
         }),
-        location: Joi.object().keys({
+        location: Joi.object().optional().keys({
```

Comment:
`author` allows `null`. I expect the console is written in a way to set `null`, not `undefined`, when the input is cleared.

Reply:
👍 Understandable, I shall update this